### PR TITLE
remove some un-used stuff

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: lib/PuppeteerSharp/ConnectOptions.cs [hunk 1]
```diff
@@ -13,12 +13,7 @@ public class ConnectOptions : IBrowserOptions, IConnectionOptions
         /// Whether to ignore HTTPS errors during navigation. Defaults to false.
         /// </summary>
         public bool IgnoreHTTPSErrors { get; set; }
-
-        /// <summary>
-        /// If set to true, sets Headless = false, otherwise, enables automation.
-        /// </summary>
-        public bool AppMode { get; set; }
```
Comment (SimonCropp) [inline lib/PuppeteerSharp/ConnectOptions.cs:20]: seems weird this isnt used?

Comment (kblok) [reply lib/PuppeteerSharp/ConnectOptions.cs:20]: We dropped that here https://github.com/hardkoded/puppeteer-sharp/pull/529/files .

Can we mark this as obsolete?

File: lib/PuppeteerSharp/Helpers/TaskHelper.cs [hunk 2]
```diff
@@ -47,22 +47,6 @@ public static class TaskHelper
...
-        /// <param name="timeoutAction">Action to be executed on Timeout.</param>
-        /// <param name="milliseconds">Milliseconds timeout.</param>
-        /// <param name="cancellationToken">Cancellation token.</param>
-        public static Task WithTimeout(
-            this Task task,
-            Func<Task> timeoutAction,
-            int milliseconds = 1_000,
-            CancellationToken cancellationToken = default)
-            => WithTimeout(task, timeoutAction, TimeSpan.FromMilliseconds(milliseconds), cancellationToken);
-
```
Comment (kblok) [inline lib/PuppeteerSharp/Helpers/TaskHelper.cs:65]: I would leave this.

File: lib/PuppeteerSharp/ConnectOptions.cs (hunk 1 above)
Comment (SimonCropp) [reply lib/PuppeteerSharp/ConnectOptions.cs:20]: fixed

File: lib/PuppeteerSharp/Helpers/TaskHelper.cs (hunk 2 above)
Comment (SimonCropp) [reply lib/PuppeteerSharp/Helpers/TaskHelper.cs:65]: done

**Pull request merged**